.travis.yml

```diff
@@ -1,5 +1,9 @@
 dist: trusty
-sudo: required
+sudo:
```

Comment:
Should there be a value here?

Reply:
I had some conflicts during merging.
Fixed.